runners/local-java/src/main/java/org/apache/beam/runners/local/Bundle.java

```diff
@@ -50,8 +50,8 @@
    * past this point before consuming this bundle.
    *
    * <p>This value is no greater than the earliest incomplete processing time or synchronized
-   * processing time {@link TimerData timer} at the time this bundle was committed, including any
-   * timers that fired to produce this bundle.
+   * processing time {@link org.apache.beam.runners.core.TimerInternals.TimerData timer} at the time
```

Comment:
I would just remove the dependency and the `link` instead of adding a dependency for javadoc only

Reply:
Done. Removed the dependency and the link